src/content/protocols/compound-v2.md

```diff
@@ -0,0 +1,338 @@
+| CErc20Delegator ( Proxy ) / CErc20 / CEther | \_setImplementation             | Upgrades the **`CErc20Delegate`** (market logic). Malicious code could steal balances or block withdrawals.                                                                                                                                                                                                                                                                                        | Timelock (DAO)                      |
++---------------------------------------------+---------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-------------------------------------+
+| CErc20Delegate                              | initialize                      | Sets all of the core parameters on the delegate implementation                                                                                                                                                                                                                                                                                                                                     | Timelock (DAO)                      |
++---------------------------------------------+---------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-------------------------------------+
+| CErc20Delegate                              | sweepToken                      | Moves tokens accidentally sent to the CToken contract to its admin, i.e., the Timelock. A rogue admin could repeatedly call sweepToken to empty out significant balances of arbitrary ERC-20 tokens held by the contract.                                                                                                                                                                          | Timelock (DAO)                      |
```

Comment:
This is meant to retrieve tokens sent by accident, is there any way to steal actual user funds from it?